packages/ui-extensions/src/utilities/shouldRenderRegistration.ts

```diff
@@ -0,0 +1,20 @@
+export interface ShouldRenderRegistrationFunction<ExtensionPoints> {
+  <Target extends keyof ExtensionPoints>(
+    target: Target,
+    callback: ExtensionPoints[Target],
+  ): ExtensionPoints[Target];
+}
+
+export function createShouldRenderRegistrationFunction<
+  ExtensionPoints,
+>(): ShouldRenderRegistrationFunction<ExtensionPoints> {
+  const extensionWrapper: ShouldRenderRegistrationFunction<ExtensionPoints> = (
+    target,
+    callback,
+  ) => {
+    (globalThis as any).shopify?.run(target, callback);
```

Comment:
`run()` seems like an odd name for this, since it is registering rather than running.

Reply:
Follow up to this change: https://github.com/Shopify/extensibility/pull/188/files